coalib/parsing/filters/LanguageFilter.py

```diff
@@ -0,0 +1,27 @@
+def language_filter(bears, languages, log_printer):
+    local_bears, global_bears = bears
```

Comment:
why the heck do we need to display bears differently depending on whether they are a `GlobalBear` or `LocalBear`... maybe we wanna change that @shreyans800755 ? But that's quite a task, so you don't have to ;)